zerver/models/realms.py

```diff
@@ -262,47 +262,17 @@ class Realm(models.Model):  # type: ignore[django-manager-missing] # django-stub
     POLICY_NOBODY = 6
     POLICY_OWNERS_ONLY = 7
```

Comment:
Would be better to remove these to make sure we store the data only in a single place. But these are used in a couple of places where the function is used for multiple settings - like in https://github.com/zulip/zulip/blob/39c6a01c7493ba55277c31ba9034ebc05dcbd9c3/zerver/models/users.py#L750 and https://github.com/zulip/zulip/blob/39c6a01c7493ba55277c31ba9034ebc05dcbd9c3/zerver/lib/test_classes.py#L1692.

Reply:
Yeah, this is fine; we'll be replacing them with the groups system anyway.